pkg/sentry/socket/epsocket/stack.go

```diff
@@ -138,3 +140,24 @@ func (s *Stack) TCPSACKEnabled() (bool, error) {
 func (s *Stack) SetTCPSACKEnabled(enabled bool) error {
 	return syserr.TranslateNetstackError(s.Stack.SetTransportProtocolOption(tcp.ProtocolNumber, tcp.SACKEnabled(enabled))).ToError()
 }
+
+// Forwarding implements inet.Stack.Forwarding.
+func (s *Stack) Forwarding(protocol tcpip.NetworkProtocolNumber) bool {
+	switch protocol {
+	case ipv4.ProtocolNumber, ipv6.ProtocolNumber:
+		return s.Stack.Forwarding(protocol)
+	default:
+		panic(fmt.Sprintf("unsupported protocol %v", protocol))
```

Comment:
Given this happens based off of input to the method, This should probably return false or return an error. In other places we have logged a warning.

Reply:
The motivation behind panic here is the following: a user can't pass the protocol variable, and it's up to a developer to pick up a protocol number. For example, the developer executes SetForwarding(arp.ARPProtocolNumber), and arp protocol doesn't support forwarding. Does it make sense to handle a returned error? Panic will help the developer to understand that her intention is wrong quickly.

Reply:
Yeah, but if a bug isn't caught until it is merged, this could be pretty disruptive. I'd log a warning and return false to be consistent.

Reply:
fixed